.changeset/dirty-houses-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"infrastructure": minor
+---
+
+[B2BP-503] - Terraform CDN for multitenancy
```

Comment:
@maurodandrea what you write here will be visible into [the CHANGELOG file](https://github.com/pagopa/b2b-portals/blob/b06ec3a6cb23a7a96d977de7505af47831e1539b/apps/infrastructure/CHANGELOG.md).

so, my question is: do you think that message is valuable to someone who will read the changelog?

Reply:
ok, i've updated the message